### PR TITLE
chore(readme): Add Code Block Syntax Highlighting

### PR DESCRIPTION
**Pull request by FlorianWendelborn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,20 +152,20 @@ class MyClass {
...
   id int primary key,
   name text
 )
-</pre>
+```
 
 Custom expressions would work without any special composition of tag template functions:
 
-<pre>
+<pre lang="javascript">
```
Comment (mmkal) [inline README.md:168]: 👍 nice I didn't know about this

File: README.md [hunk 2]
```diff
@@ -44,14 +44,14 @@ class MyClass {
 
 This logs:
 
-<pre>
+```sql
 
       create table student(
         id int primary key
         name text
       )
```
Comment (jridgewell) [inline README.md:52]:
```suggestion
create table student(
  id int primary key
  name text
)
```

Comment (mmkal) [reply README.md]: @jridgewell this code block is supposed to show that the output has an unwanted indentation - it shouldn't be "fixed" like this. I think we should revert this edit. Unless I'm missing something?

Comment (jridgewell) [reply README.md]: Whoops. I'll submit a PR to fix.

**Pull request merged**